src/Components/CreateImageWizard/CreateImageWizard.js

```diff
@@ -606,6 +611,9 @@ const CreateImageWizard = () => {
           {
             component: componentTypes.WIZARD,
             name: 'image-builder-wizard',
+            ModalProps: {
+              appendTo,
```

Comment:
This feels not great, but I guess there's no way around it currently? Why exactly does it break otherwise?

Reply:
sure

Reply:
updated